.github/workflows/screenshot-tests.yaml

```
@@ -0,0 +1,42 @@
+name: Run screenshot tests
+
+on:
+  workflow_call
+
+jobs:
+  screenshot-tests:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    container: ghcr.io/netcracker/qubership-apihub-nodejs-dev-image:1.7.3
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          registry-url: 'https://npm.pkg.github.com/'
+
+      - name: Install dependencies
+        run: npm ci
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run tests
+        id: tests
+        continue-on-error: true
+        run: npm run screenshot-test:ci --if-present
```

Let's remove --if-present here
We want to use this workflow only for the repositories where we do have screenshot test and given that, it is better to have fail fast approach here

---

Done